spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,18 @@
+# spec/features/user_show_spec.rb
+
+RSpec.describe 'UserShow', type: :feature do
+  before do
+    @user = User.create!(name: 'User', email: 'user@example.com')
+    visit user_path(@user)
+  end
+
+  it 'shows the user details and their posts' do
+    expect(page).to have_content(@user.name)
+    expect(page).to have_content("Number of posts: #{@user.posts.count}")
+  end
+
+  it 'redirects to the full post list on click' do
+    click_on('See all posts', match: :first)
+    expect(page).to have_current_path(user_path(@user))
+  end
```

Comment:
- [ ] Just as I have pointed previously, this is also required to have a total test of 8 but you have only two test written, kindly add the rest of the test.
![image](https://github.com/jlberbesi/blog-app/assets/86200356/b5dc2682-18c2-475f-85db-98161e1bde85)

Reply:
![imagen](https://github.com/jlberbesi/blog-app/assets/88105477/e37e57c6-6e52-44e2-9690-d2f8f297cf4b)